### DISCUSSION TITLE
How to set binary mode font?

### DISCUSSION THREAD

**Discussion by HubKing** [WinMerge/winmerge#952] [Q&A]
I can Ctrl+wheel to increase the font temporarily, but how do I change the default font size for the binary mode?

Also, if possible, can I increase the font of the UI itself? Like the main menu, the Options dialogue, the file name tab, the status bar, etc. The UI looks smaller than I want. Please do not suggest "Just increase the system DPI setting".

**Comment by sdottaka** [comment]
You can specify the font size in the Options dialog (Compare/Binary category → Character Set... → Font size in points) 

![image](https://user-images.githubusercontent.com/98126/132110387-e24c3ce0-e0d0-4917-b906-bcde2b06b0db.png)

**Comment by stanio** [comment]
> You can specify the font size in the Options dialog

Could the font family be customized as well?  I have chosen a font for standard text diffs (View / Select Font...) I would like to see in the binary diffs as well.

**Reply by sdottaka** [reply]
Unfortunately, in the Binary compare window(frhed), the font is fixed to Lucida Console.

**Reply by stanio** [reply]
Thank you for the clarification.
